### PR TITLE
Negadoctor improvements

### LINKED ISSUE #97: Feedback on negadoctor
From pixls.us, on the negadoctor entry in dtdocs:

I think it’s a great first draft. But I have a suggestion for improvement:

It would be helpful if explicit instructions were included for the parts of the image to sample when using the eye droppers. It is already clear for the colour of the film base, D-max, shadows color cast, and paper black sliders. It is not so clear for the scan exposure bias, highlights white balance and print exposure adjustment sliders.

For the scan exposure bias slider, it currently says “Again, you can use the eye dropper to allow Negadoctor to automatically calculate any needed offset.” Are we to assume we should be sampling the whole image minus the unexposed part?
For the highlights white balance slider, it currently says “For highlight color casts, select the eye dropper and click and drag a rectangle across the brightest area of your image.” Does it matter if the brightest part of the image has a colour cast? Should we be aiming for a neutral white, or just any bright part (bearing in mind that the brightest part could be green, blue, yellow, etc.)?
For the print exposure adjustment slider, there is currently no guidance for how to use the eye dropper. Do we assume we should be sampling the whole image minus the unexposed part?

The only other issue I have is that I don’t fully understand the real-world differences between paper black, scan exposure bias, paper grade and print exposure adjustment. The manual does a fairly decent job of explaining what their purpose is and what they are trying to achieve, but when actually using the tool, it appears as though they are all applying slightly different tone curves and that there’s some duplication. I often find that after going through the last tab, the image looks hardly any different from after completing the first two tabs. Not sure if that’s really something the manual can expand upon.

### PR DESCRIPTION
**Pull request by matt-maguire**
Fixes #97 

add some clarifications to negadoctor arising out of pixls.us duscussion.

### CODE REVIEW

**Comment by elstoc** [comment]
Thanks.

File: content/module-reference/processing-modules/negadoctor.md [hunk 1]
```diff
@@ -64,17 +72,22 @@ The settings on this tab should not be needed for most well-preserved negatives
...
 
 ## print properties
 
-This tab contains settings that will mimic the tonal effect of the photochemical papers that would have been used to create the hard copy image if you were not developing the photo digitally.
+This tab contains settings that will mimic the tonal effect of the photochemical papers that would have been used to create the hard copy image if you were not developing the photo digitally. These are creative settings, and serve a similar overall purpose to the creative tone curve settings on the _look_ tab of the [_filmic rgb_](filmic-rgb.md) module.
+
+It can be noted that the _print exposure_, _paper black_, _paper grade_ and _print exposure_ are analogous to the _slope_, _offset_ and _power_ controls in the [_color balance_](color-balance.md) module (which in turn is based on the ACES CDL standard). They bake a creative tone curve to enforce your contrast intent after the inversion, at the end of the module. The equation governing this slope/offset/power behaviour is:
```
Comment (aurelienpierre) [inline content/module-reference/processing-modules/negadoctor.md:84]: the color balance is based on the ASC (American society of cinematographers) CDL (color decision list). ACES is something else.

Comment (elstoc) [reply content/module-reference/processing-modules/negadoctor.md:84]: Thanks @aurelienpierre. I'll correct it.

Comment (elstoc) [reply content/module-reference/processing-modules/negadoctor.md:84]: done

**Pull request merged**